src/waves.ts

```diff
@@ -8,43 +9,34 @@ export const fetchWaves = async (
   ) => void,
   setActiveWaveIndex: (index: number) => void
 ) => {
-  const wavesData: RawData[][] = await fetch(
-    "https://d21fj0gildolug.cloudfront.net/load_mentors"
-  )
+  const { mentorsUrl } = config;
+  const mentorsData: RawData[] = await fetch(mentorsUrl)
     .then((response) => response.json())
-    .then(({ waves }) => waves);
+    .then(({ mentors }) => mentors);
 
-  const formatMentor = (mentor: RawData): Mentor => ({
-    courseOfStudy: mentor["Course of Study"],
-    fullBio: mentor.Biography,
-    fullImageUrl:
-      Array.isArray(mentor.Photo) && mentor.Photo.length > 0
-        ? mentor.Photo[0].url
-        : "/mentor-thumbnail.png",
-    industries: [
-      mentor["Industry 1"] ?? [],
-      mentor["Industry 2"] ?? [],
-      mentor["Industry 3"] ?? [],
-    ].flat(),
-    name: mentor.Name,
-    organisation: mentor.Organisation,
-    role: mentor["Job Title"],
-    school: mentor.School,
-    thumbnailImageUrl:
-      Array.isArray(mentor.Photo) &&
-      mentor.Photo.length > 0 &&
-      mentor.Photo[0].thumbnails
-        ? mentor.Photo[0].thumbnails.large.url
-        : "/mentor-thumbnail.png",
+  const createMentor = (mentor: RawData): Mentor => ({
+    courseOfStudy: mentor.course_of_study,
+    fullBio: mentor.full_bio,
+    fullImageUrl: mentor.full_image_url,
+    industries: mentor.industries,
+    name: mentor.name,
+    organisation: mentor.organisation,
+    role: mentor.role,
+    school: mentor.school,
+    thumbnailImageUrl: mentor.thumbnail_image_url,
   });
 
-  const waves: { [key: string]: Mentor }[] = [];
+  const numWaves =
+    Math.max(...mentorsData.map((mentorData) => mentorData.wave_id)) + 1;
```

Comment:
@wei2912 

Encountered this error... not reading mentors data proper.

![image](https://user-images.githubusercontent.com/66960654/147893761-f4148611-18ff-414c-8301-10329560f9be.png)

Reply:
This error occurs as there were changes to the main code which require the Cloudfront URL to return the new JSON format. I can get the lambda uploaded for you to test the code, but this will temporarily break the main website.